<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ccattitude</class>
 <widget class="QWidget" name="ccattitude">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>110</height>
      </size>
     </property>
     <property name="title">
      <string>Attitude Rotation</string>
     </property>
     <widget class="QSpinBox" name="rollBias">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="pitchBias">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-90</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="yawBias">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Roll</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pitch</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Yaw</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>341</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select the amount to rotate the attitude</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>140</height>
      </size>
     </property>
     <property name="title">
      <string>Attitude Calibration</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>371</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Place aircraft flat before computing</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="status">
           <property name="text">
            <string>Click &quot;Zero Accel Bias&quot; to start</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="zeroBias">
           <property name="toolTip">
            <string>Launch horizontal calibration.</string>
           </property>
           <property name="text">
            <string>Zero Accel Bias</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="zeroBiasProgress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>groupBox_2</zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="toolTip">
        <string>Click to permanently save the accel bias in the CopterControl Flash.</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
